src/core/metrics.cc

```diff
@@ -367,6 +368,14 @@ Metrics::GetRegistry()
   return singleton->registry_;
 }
 
+const std::string
+Metrics::SerializedMetrics()
+{
+  auto singleton = Metrics::GetSingleton();
+  return singleton->serializer_->Serialize(
```

Comment:
Is there a serializer that doesn't include all the comments in the outout?

Reply:
No, this is the only serializer provided by prometheus-cpp and there is no option to exclude the comments. To do so, we can either strip the comments out from the serialized metrics or provide our own serializer implementation.